adafruit_airlift/esp32.py

```diff
@@ -89,7 +97,7 @@ def __init__(
         # Used for WiFi mode.
         self._spi = spi
 
-    def reset(self, mode, debug=False):
+    def reset(self, mode: int, debug: bool = False):
```

Comment:
```suggestion
    def reset(self, mode: int, debug: bool = False) -> None:
```

Reply:
If you don't have the `-> None`, it reverts to `-> Any`, so it's better to include it explicitly. (Same in other libraries.)

Reply:
Ah so returns (and lack thereof) should always type hint as `None`?

Reply:
Yes. There are examples of that in the regular doc, for instance in `DigitalInOut`:
![image](https://user-images.githubusercontent.com/2847802/141485602-9436904a-54bd-4e48-98d8-718c1fb22e0e.png)

Reply:
Awesome, thanks for letting me know!